### PR TITLE
Docs  image and tabs extended paredown

### PR DESCRIPTION
**Pull request by RichKummer**
### Related Ticket(s)

#1125 

### Description

Updates to Images page and nav title, animated gif, and "Resources" header on each component page. Includes missed updates to links to Carbon design system URLs.

### Changelog

**New**

- Updated Image with caption animation

**Changed**

- Image page -> Images page.
- Image nav -> Images nav.
- Component pages with only one component have Resources as an H2.
- Updated Overview language on Images.

**Removed**

- old version of animation gif.
- H2/3 in Ari's ResourceLinks file

### CODE REVIEW

**Comment by shixiedesign** [comment]
Just 1 small fix: 

In `Tabs extended > Overview`

> The tab labels may wrap up to two lines when truncated.

How about "The tab labels may wrap up to two lines before it gets truncated. Truncated labels are displayed in tooltips on hover and focus."

Otherwise this looks great!

File: src/pages/components/images.mdx [hunk 1]
```diff
@@ -72,7 +73,7 @@ Inside the overlay, the caption becomes the heading and there is an opportunity
 
 </Column>
 </Row>
-<Caption>When an image has a caption, the user can hover over the image and click to trigger a Lightbox overlay with more information.</Caption>
+<Caption>When an image has a caption, the user can hover over the image and click to trigger a lightbox overlay with more information.</Caption>
```
Comment (janchild) [inline src/pages/components/images.mdx:76]:
```suggestion
<Caption>Users can hover over the image and click to open a lightbox overlay that includes a larger image and more information.</Caption>
```

**Review by janchild — COMMENTED** [review]
One tweak to the caption. The rest looks great!! Looking forward to seeing the final build! 😄

**Review by janchild — APPROVED** [review]
🥂

**Pull request closed without merging**